003-ws-in-core.md

```diff
@@ -0,0 +1,30 @@
+| Title  | ws in core                  |
+|--------|-----------------------------|
+| Author | @eljefedelrodeodeljefe      |
+| Status | DRAFT                       |
+| Date   | 2015-01-25 02:34:00         |
+
+## Description
+
+This EP proposes having the current [websockets/ws](https://github.com/websockets/ws) userland implementation in core.
+Efforts have already been made on nodejs/node#4842. It is understood that
+adaptions to style and simplicity need to be done in order to conform with
+`node` core.
+
+## Motivation
+
+> `websockets` are the primary reason for companies, which have **not** decided
+to make a commitment towards `node` to use `node` anyways. For a wider adoption
+of the language this is a critical point.
```

Comment:
If companies are already adopting node.js because of how good the `ws` package is, then that means the `ws` maintainers are doing a great job and there's nothing that node core needs to do.